app/controllers/initiatives_controller.rb

```diff
@@ -27,7 +33,6 @@ def edit
 
   def update
     if @initiative.update(initiative_params)
-      populate_initiative_sets
       redirect_to root_path
```

Comment:
After you `create` you redirect to `:initiatives`. What is `root_path`? Is it the same as `:initiatives`?

Reply:
Same. changed to root_path